Game/Scripts/PlayerManagerScript.cpp

```diff
@@ -90,6 +98,19 @@ PlayerMoveScript* PlayerManagerScript::GetMovementManager() const
 	return movementManager;
 }
 
+void PlayerManagerScript::ForcingJump(bool forcedJump)
```

Comment:
```suggestion
void PlayerManagerScript::TriggerJump(bool forcedJump)
```
just a matter of personal preference, but don't you find this name easier to understand?